### PR TITLE
Remove deprecated function calls

### PR DESCRIPTION
**Pull request by SimonBaeumer**
Removed method calls on deprecated methods.
- Set minimum version to `5.2.0`
- Usage of the new attribute system in `Setup.php` & `Uninstall.php`
- Removed `Shopware()->Bootstrap` calls
- Removed `Enlight()` calls with `Shopware()`

**Todo:**
- `engine/Shopware/Plugins/Local/Backend/SwagConnect/Bootstrap/Setup.php:610`

### CODE REVIEW

File: Tests/Shopware/Connect/Component/ImportServiceTest.php [hunk 1]
```diff
@@ -54,7 +54,7 @@ public function setUp()
         $this->importService = new ImportService(
             $this->manager,
             Shopware()->Container()->get('multi_edit.product'),
-            $this->remoteCategoryRepository,
+            $this->categoryRepository,
```
Comment (SimonBaeumer) [inline Tests/Shopware/Connect/Component/ImportServiceTest.php]: @svenfinke This crashed on php7

**Comment by SimonBaeumer** [comment]
@svenfinke The `billing` and `shipping` models will be available until Shopware 5.4, so it is not needed to replace the usage of them. 

The only  deprecation which must be fixed is the `\Shopware\Models\Article\Element`

**Comment by mkafadarov** [comment]
@SimonBaeumer, @sbofirov fixed all tests for php7 so you can see his change over here:
https://github.com/shopware/SwagConnect/pull/191

**Comment by sbofirov** [comment]
The changes from this PR are already applied or they will be with following PR https://github.com/shopware/SwagConnect/pull/238

**Pull request closed without merging**